### PR TITLE
Update join

### PR DESCRIPTION
**Pull request by cmontella**

### CODE REVIEW

**Comment by cmontella** [comment]
Fixes https://github.com/witheve/docs/issues/36

File: handbook/math/range.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ bind @browser
   [#div sort: y, text: y]
 ```
 
-Generate and display the odd integers between 1 and 10. In this example, `y = {1, 3, 5, 6, 9}`. Notice the set does not include `10` in this case.
+Generate and display the odd integers between 1 and 10. In this example, `y = (1, 3, 5, 7, 9)`. Notice the set does not include `10` in this case.
```
Comment (convolvatron) [inline handbook/math/range.md:43]: it occurs to me that maybe {...} is a more generally appreciated notation for sets than (...)

**Review by convolvatron — COMMENTED** [review]
👍

**Review by convolvatron — APPROVED** [review]
total, absolute and unconditional approval

**Pull request merged**